From: Thomas Moulard <thomas.moulard@gmail.com>
Date: Fri, 12 Jul 2013 16:10:46 +0900
Subject: Fix compilation error on FreeBSD

---
 src/tools/io/vpKeyboard.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/tools/io/vpKeyboard.cpp b/src/tools/io/vpKeyboard.cpp
index 429e6c5..c79eb43 100644
--- a/src/tools/io/vpKeyboard.cpp
+++ b/src/tools/io/vpKeyboard.cpp
@@ -136,10 +136,10 @@ vpKeyboard::setRawMode(bool active)
     new_settings.c_lflag &= (unsigned int)~ICANON;
     new_settings.c_lflag &= (unsigned int)~ECHO;
     new_settings.c_lflag &= (unsigned int)~ISIG;
-    new_settings.c_oflag &= (unsigned int)~NL0;
-    new_settings.c_oflag &= (unsigned int)~CR0;
+    //new_settings.c_oflag &= (unsigned int)~NL0;
+    //new_settings.c_oflag &= (unsigned int)~CR0;
     new_settings.c_oflag &= (unsigned int)~TAB0;
-    new_settings.c_oflag &= (unsigned int)~BS0;
+    //new_settings.c_oflag &= (unsigned int)~BS0;
     new_settings.c_cc[VMIN] = 1;
     new_settings.c_cc[VTIME] = 0;
     tcsetattr(STDIN_FILENO, TCSANOW, &new_settings);
